setup.py

```diff
@@ -5,7 +5,7 @@
 requirements = [
     "galaxy_importer==0.1.3b8",
     "packaging",
-    "pulpcore-plugin~=0.1rc6",
+    "pulpcore~=3.0rc7",
```

Comment:
I had expected 3.0.0rc8

Reply:
The second '0' is important with the ~= because otherwise this is declaring compatability also with 3.1 is my understanding. Also we need rc8 since that has the plugin API in it.

Reply:
I don't think that's the case with pre-release versions but I could be wrong.  I can bump the version number though and double-check my assumption ^^

Reply:
The problem with depending on the later package is that installs fail since it's not yet availalbe on pypi...

```
stdout: Obtaining file:///home/vagrant/devel/pulp_ansible
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'done'
    Preparing wheel metadata: started
    Preparing wheel metadata: finished with status 'done'
Requirement already satisfied: PyYAML in /usr/local/lib/pulp/lib/python3.7/site-packages (from pulp-ansible==0.2.0b6.dev0) (5.1.2)
Collecting galaxy-importer==0.1.3b8 (from pulp-ansible==0.2.0b6.dev0)
  Downloading https://files.pythonhosted.org/packages/1f/54/66864da3e8587e1fb68f43765c9c895c714eea7e8333a2352a604478ff80/galaxy_importer-0.1.3b8-py3-none-any.whl
Collecting semantic-version (from pulp-ansible==0.2.0b6.dev0)
  Downloading https://files.pythonhosted.org/packages/0f/3b/8fee26649a86c71df159ed0ae7ac5f9ac38829bccd8a7404e116f903929b/semantic_version-2.8.2-py2.py3-none-any.whl
Collecting pulpcore~=3.0.0rc8 (from pulp-ansible==0.2.0b6.dev0)

:stderr:   Could not find a version that satisfies the requirement pulpcore~=3.0.0rc8 (from pulp-ansible==0.2.0b6.dev0) (from versions: 3.0.0a1.dev3, 3.0.0a1.dev4, 3.0.0a1.dev5, 3.0.0a1, 3.0.0a2, 3.0.0a3, 3.0.0a4, 3.0.0a5, 3.0.0a6, 3.0.0a7, 3.0.0a8, 3.0.0a9, 3.0.0a10, 3.0.0a11, 3.0.0a12, 3.0.0a13, 3.0.0a14, 3.0.0a15, 3.0.0a16, 3.0.0a17, 3.0.0a18, 3.0.0a19, 3.0.0a20, 3.0.0a21, 3.0.0a22, 3.0.0a23, 3.0.0a24, 3.0.0a25, 3.0.0a26, 3.0.0a27, 3.0.0a28, 3.0.0a29, 3.0.0b1, 3.0.0b2, 3.0.0b3, 3.0.0b4, 3.0.0b5, 3.0.0b6, 3.0.0b7, 3.0.0b8, 3.0.0b9, 3.0.0b10, 3.0.0b11, 3.0.0b12, 3.0.0b13, 3.0.0b14, 3.0.0b15, 3.0.0b16, 3.0.0b17, 3.0.0b18, 3.0.0b19, 3.0.0b20, 3.0.0b21, 3.0.0b22, 3.0.0b23, 3.0.0rc1, 3.0.0rc2, 3.0.0rc3, 3.0.0rc4, 3.0.0rc5, 3.0.0rc6, 3.0.0rc7)
No matching distribution found for pulpcore~=3.0.0rc8 (from pulp-ansible==0.2.0b6.dev0)
```

Reply:
Actually with this PR it works for source checkouts.